src/main/java/org/fcrepo/migration/handlers/ocfl/ArchiveGroupHandler.java

```diff
@@ -148,68 +137,57 @@ public void processObjectVersions(final Iterable<ObjectVersionReference> version
                 final String dsId = dv.getDatastreamInfo().getDatastreamId();
                 final String f6DsId = resolveF6DatastreamId(dsId, f6ObjectId, mimeType);
                 final var datastreamFilename = lastPartFromId(f6DsId);
-                final var contentPath = PersistencePaths.binaryContentPath(datastreamFilename);
 
                 if (dv.isFirstVersionIn(ov.getObject())) {
                     dsCreateDates.put(dsId, dv.getCreated());
                 }
                 final var createDate = dsCreateDates.get(dsId);
 
                 final var datastreamHeaders = createDatastreamHeaders(dv, f6DsId, f6ObjectId,
-                        datastreamFilename, mimeType, createDate, contentPath);
+                        datastreamFilename, mimeType, createDate);
 
                 if ("RE".contains(dv.getDatastreamInfo().getControlGroup())) {
```

Comment:
Could you add a comment explaining the purpose of this line?  Also perhaps the above "R" and "E" map keys should be made into constants and this line should reference them?  Just a suggestion for clarity.

Reply:
I did not write that code nor interact with it in this PR, but, yes, I can clean it up.